src/archetypes/java-web-app/build.ts

```diff
@@ -0,0 +1,33 @@
+const BasicBuilder = require.main?.exports.BasicBuilder as any
+const path = require('path')
+
+const MyBuilder = class extends BasicBuilder {
+  // eslint-disable-next-line no-useless-constructor
+  constructor(settings: any) {
+    super(settings)
+  }
+
+  processTemplates(oc: any) {
```

Comment:
Could this break if it receives an `oc` that doesn't respond to `.git`?

Reply:
Discussed on the team call.

Reply:
That is because we are importing things from `nr-pileline-ext` which does not use typescript. The goal is to eventually have `nr-pipeline-ext` moved to nrdk or at least converted to typescript.